packages/editor-ui/src/views/SettingsUsageAndPlan.vue

```diff
@@ -1,44 +1,159 @@
+		await usageStore.refreshLicenseManagementToken();
+	} else {
+		await usageStore.getLicenseInfo();
+	}
+	usageStore.setLoading(false);
+});
+
+watch(() => usageStore.error, (error: UsageState['error']) => {
+	if(error?.message) {
+		Notification.error({
+			title: 'Error',
+			message: error.message,
+			position: 'bottom-right',
+		});
+	}
 });
 
+watch(() => usageStore.success, (success: UsageState['success']) => {
+	if(success?.title && success?.message) {
+		Notification.success({
+			title: success.title,
+			message: success.message,
+			position: 'bottom-right',
+		});
+	}
+});
+
+const sendUsageTelemetry = (action: UsageTelemetry['action']) => {
+	const telemetryPayload = usageStore.telemetryPayload;
+	telemetryPayload.action = action;
+	telemetry.track('User clicked button on usage page', telemetryPayload);
+};
+
+const onAddActivationKey = () => {
+	activationKeyModal.value = true;
+	sendUsageTelemetry('add_activation_key');
+};
+
+const onViewPlans = () => {
+	sendUsageTelemetry('view_plans');
+};
+
+const onManagePlan = () => {
+	sendUsageTelemetry('manage_plan');
+};
+
+const onDialogClosed = () => {
+	activationKey.value = '';
+};
+
+const onDialogOpened = () => {
+	activationKeyInput.value?.focus();
+};
+
 </script>
 
 <template>
-	<div  v-if="!usageStore.isLoading">
+	<div v-if="!usageStore.isLoading">
 		<n8n-heading size="2xlarge">{{ $locale.baseText('settings.usageAndPlan.title') }}</n8n-heading>
 		<n8n-heading :class="$style.title" size="large">
-			{{ $locale.baseText('settings.usageAndPlan.plan', { interpolate: {plan: usageStore.planName } } ) }}
+			<i18n path="settings.usageAndPlan.description">
+				<template #name>{{ usageStore.planName }}</template>
+				<template #type>
+					<span v-if="usageStore.planId">{{ $locale.baseText('settings.usageAndPlan.plan') }}</span>
+					<span v-else>{{ $locale.baseText('settings.usageAndPlan.edition') }}</span>
+				</template>
+			</i18n>
 		</n8n-heading>
 		<div :class="$style.quota">
 			<n8n-text size="medium" color="text-light">{{ $locale.baseText('settings.usageAndPlan.activeWorkflows') }}</n8n-text>
-			<i18n :class="$style.count" path="settings.usageAndPlan.activeWorkflows.count">
-				<template #count>{{ usageStore.executionCount }}</template>
-				<template #limit>
-					<span v-if="usageStore.executionLimit < 0">{{ $locale.baseText('_reusableBaseText.unlimited') }}</span>
-					<span v-else>{{ usageStore.executionLimit }}</span>
-				</template>
-			</i18n>
+			<div :class="$style.chart">
+				<span v-if="usageStore.executionLimit > 0" :class="$style.line">
+					<span :class="$style.bar" :style="{ width: `${usageStore.executionPercentage}%` }"></span>
+				</span>
+				<i18n :class="$style.count" path="settings.usageAndPlan.activeWorkflows.count">
+					<template #count>{{ usageStore.executionCount }}</template>
+					<template #limit>
+						<span v-if="usageStore.executionLimit < 0">{{ $locale.baseText('_reusableBaseText.unlimited') }}</span>
```

Comment:
we are not supposed to use reusable texts directly.. that's why TS is complaining